frc_characterization/logger_analyzer/data_analyzer.py

```diff
@@ -962,26 +962,33 @@ def defineTestResults(*args):
         trackWidthEntry.configure(state="disabled")
         kGEntry.configure(state="disabled")
         kCosEntry.configure(state="disabled")
-        if STATE.test.get() != "Drivetrain":
+        if STATE.test.get() != Tests.DRIVETRAIN.value:
```

Comment:
```suggestion
		test = Tests(STATE.test.get())
        if test != Tests.DRIVETRAIN:
```
ditto for lines 972, 983

Reply:
[Done](https://github.com/Piphi5/frc-characterization/blob/Refactor/frc_characterization/logger_analyzer/data_analyzer.py#L969)